pkg/controller/cluster/cluster.go

```diff
@@ -413,3 +426,76 @@ func (c *ClusterReconciler) validate(cluster *v1alpha1.Cluster) error {
 	}
 	return nil
 }
+
+// lookupServiceCIDR will try to lookup the serviceCIDR of the cluster.
+// It will first look for the 'kube-apiserver' pod, looking for the --service-cluster-ip-range flag.
+// Otherwise
+func (c *ClusterReconciler) lookupServiceCIDR(ctx context.Context) (string, error) {
+	log := ctrl.LoggerFrom(ctx)
+
+	// find the kube-apiserver pod, and look for the '--service-cluster-ip-range' flag
+	matchingLabels := ctrlruntimeclient.MatchingLabels(map[string]string{
```

Comment:
we may as well just use the fallback method, since that not all distros (host clusters) run the api server as a static pod

Reply:
Gotcha!